dictionaries/git.json

```diff
@@ -11,7 +11,7 @@
 "ST*TS": "status",
 "TKPW*EUTS": "{^}git status",
 "TKPW*TS": "{^}git status{#Return}",
-"HROG/H*PB/P*": "log -p",
```

Comment:
This builds on the convention for `"H*PB": "{-^}",` set up in the [code dictionary](https://github.com/didoesdigital/steno-dictionaries/blob/master/dictionaries/code.json#L73)

Reply:
Okay, I'll remove the commit that changes single hyphen flag usage from `H*PB` to `H-PBS`.